src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/SwapGrTags.hpp

```diff
@@ -0,0 +1,51 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/VariablesTag.hpp"
```

Comment:
unused

Reply:
It is used by `typename System::spacetime_variables_tag` :shrug:

Reply:
then it should be in the cpp file, not here

Reply:
We usually include the tag files in the header for tags that are used in the header because otherwise it's unclear who defines the tag, no?